.circleci/docker/common/install_cache.sh

```diff
@@ -8,7 +8,11 @@ sed -e 's|PATH="\(.*\)"|PATH="/opt/cache/bin:\1"|g' -i /etc/environment
 export PATH="/opt/cache/bin:$PATH"
 
 # Setup compiler cache
-curl --retry 3 https://s3.amazonaws.com/ossci-linux/sccache -o /opt/cache/bin/sccache
+if [ -n "$ROCM_VERSION" ]; then
+  wget -O /opt/cache/bin/sccache http://repo.radeon.com/misc/.sccache_amd/sccache
```

Comment:
Please use `curl` instead of `wget`
```suggestion
  curl --retry 3 http://repo.radeon.com/misc/.sccache_amd/sccache -o /opt/cache/bin/sccache 
```

Reply:
I will update to the curl command.